41-vaccine-temp-monitor/firmware/arduino/src/main.cpp

```diff
@@ -0,0 +1,528 @@
+
+/*
+ *                          |   LM75A   |
+ *                          |           |
+ *                          | V G S S   |
+ *                          | C N D C O |
+ *                          |_C_D_A_L_S_|
+ * __________________,        | | | | |        ,__________________
+ *       ESP32       |        | | | | |        |   NOTECARRIER-B
+ *                   |        | | | | |        |
+ *                   |        | | | | '--------|AUX1
+ *                   |        | | | |          |
+ *                   |        '-^-^-^----------|<VIO
+ *                   |          | | |          |
+ *               VBAT|----------^-^-^----------|VBAT
+ *                   |          | | |          |
+ *                GND|----------'-^-^----------|GND
+ *                   |            | |          |
+ *                SDA|------------'-^----------|SDA
+ *                   |              |          |
+ *                SCL|--------------'----------|SCL
+ *                   |                         |
+ *                 EN|-------------------------|ATTN
+ *                   |                         |
+ */
+
+#include <M2M_LM75A.h>
+#include <Notecard.h>
+#include <NotecardEnvVarManager.h>
+#ifdef ARDUINO_ARCH_ESP32
+#include <NotecardAuxiliaryWiFi.h>
+#endif // ARDUINO_ARCH_ESP32
+
+// Uncomment this line and replace com.your-company:your-product-name with your
+// ProductUID.
+// #define PRODUCT_UID "com.your-company:your-product-name"
+
+#ifndef PRODUCT_UID
+#define PRODUCT_UID "" // "com.my-company.my-name:my-project"
+#pragma message "PRODUCT_UID is not defined in this example. Please ensure your Notecard has a product identifier set before running this example or define it in code here. More details at https://bit.ly/product-uid"
+#endif
+#define usbSerial Serial
+#define txRxPinsSerial Serial1
+
+#define CRT_MAX_TEMP_C 30

```

Comment:
What's CRT?

Reply:
You don't know!? I heard they were teaching it in schools!

Reply:
Code comment moved in accordance with your subsequent review comment.